manifests/init.pp

```diff
@@ -19,17 +21,14 @@
     fail('cgroups::package_name must be a string or an array.')
   }
 
+  validate_hash($mounts)
+  validate_hash($groups)
+
+
   case $::osfamily {
     'RedHat': {
-      case $::operatingsystemmajrelease {
-        '6': {
-          $default_package_name   = 'libcgroup'
-          $default_cgconfig_mount = '/cgroup'
-        }
-        default: {
```

Comment:
and keep the default fail. That way if you include this on EL5 or some other unsupported RedHat like system, it will fail.